conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -440,7 +395,7 @@ def generate(self):
         # Current directory is the generators_folder
         generator_files = self.content
         for generator_file, content in generator_files.items():
-            save(generator_file, content)
+            save(self._conanfile, generator_file, content)
```

Comment:
It is valid to use the internal ``save(file, contents)`` and not the ``conan.tools`` one